### ISSUE TITLE
关于TinyChart数据可用性的问题

### ISSUE THREAD

**Issue by lucasjinreal** [X-PLUG/mPLUG-DocOwl#87]
看起来tinyChart里面包含了很多 <commetn>之类的控制符，且输出仅为numpy的代码，这种数据理论上推理也只是给一个numpy的代码，能否直接进行QA呢

**Comment by zhangliang-04** [comment]
> 看起来tinyChart里面包含了很多 之类的控制符，且输出仅为numpy的代码，这种数据理论上推理也只是给一个numpy的代码，能否直接进行QA呢

你好，tinychart可以直接进行QA，只需要把prompt中的’Answer with detailed steps‘ 去掉模型就会产生直接回答。
另外，对于如何运行模型产生的numpy代码并得到答案，请参考[parse_model_output](https://github.com/X-PLUG/mPLUG-DocOwl/blob/main/TinyChart/tinychart/eval/eval_metric.py#L53)和[evaluate_cmds](https://github.com/X-PLUG/mPLUG-DocOwl/blob/main/TinyChart/tinychart/eval/eval_metric.py#L34)这两个函数

**Comment by lucasjinreal** [comment]
不是 我的意思是，详细回答的时候，会直接输出np的代码，为啥不把分析结果显示的表示出来呢，这的数据能让模型真正benifit吗

**Comment by zhangliang-04** [comment]
你好，当然可以把每一步的执行结果放入数据里让模型学习，这是一个很好的想法！
不过我们在设计pot的时候主要的出发点是缓解计算错误，模型只需要学推理（numpy代码以及注释），计算交给解释器去做，所以每一步的结果不是必要的。

**Comment by lucasjinreal** [comment]
感谢回答，如果我把那些 <comment> 这样字段去掉，只保留numpy的代码，同时直接给出结果。模型是不是更好学一点啊

**Comment by zhangliang-04** [comment]
> 感谢回答，如果我把那些 这样字段去掉，只保留numpy的代码，同时直接给出结果。模型是不是更好学一点啊

控制字符是为了更方便地解析模型产生的代码，去掉可能会增加解析失败的频率。
我们目前没有尝试加入执行结果的pot，可以尝试一下~

**Comment by feihuamantian** [comment]
https://github.com/X-PLUG/mPLUG-DocOwl/issues/102
